src/plugins/chart/ChartRender.tsx

```
@@ -125,23 +137,129 @@ export const ChartRender: React.FC<{
   onColumnLengthChange?: (value: number) => void;
+  dataTime?: string;
 }> = (props) => {
   const handle = useFullScreenHandle() || {};
   const [chartType, setChartType] = useState<
-    'pie' | 'bar' | 'line' | 'column' | 'area' | 'descriptions' | 'table'
-  >(() => props.chartType);
+    'pie' | 'bar' | 'line' | 'column' | 'area' | 'radar' | 'scatter' | 'descriptions' | 'table'
+  >(() => props.chartType as any);
   const {
     chartData,
     node,
```

ESLint

[error] 'node' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)